.github/workflows/test-suite.yml

```diff
@@ -17,6 +17,10 @@ env:
   PINNED_NIGHTLY: nightly-2023-04-16
   # Prevent Github API rate limiting.
   LIGHTHOUSE_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  # Use self-hosted runners for the canonical sigp repo.
+  SELF_HOSTED_RUNNERS: ${{ github.repository == 'sigp/lighthouse' }}
+  RUNNER_LINUX_SMALL: ${{ vars.SELF_HOSTED_RUNNERS && '[self-hosted, linux, small]' || 'ubuntu-latest'  }}
```

Comment:
I think the `vars` context is for the config variables defined in the GitHub project, maybe we need `env` here instead? I could be wrong though.

Reply:
Yeah, thanks. There's a [some complications](https://stackoverflow.com/questions/72633058/parameterize-runs-on-value-for-github-action) about if we can use `env` variables in `runs_on` or not.

I'm going to try `SELF_HOSTED_RUNNERS` as a repository-level variable. If it works it's probably better form anyway; it allows and repo to toggle it on/off.